File: Iniciante/URI 1001.scala
import java.util.Scanner

object Main {

    def main(args: Array[String]) {
        val leitor = new Scanner(System.in)
        val a = leitor.nextInt()
        val b = leitor.nextInt()
        val x = a + b
        println("X = " + x)
    }
	
}

File: Iniciante/URI 1001.kt
import java.util.Scanner;

fun main(args: Array<String>) {
    val leitor = Scanner(System.`in`);
    
    val a: Int = leitor.nextInt();
    val b: Int = leitor.nextInt();
    val x: Int = a + b;
    
    println("X = " + x);
}